ui/component/header/view.jsx

```diff
@@ -5,11 +5,12 @@ import * as SETTINGS from 'constants/settings';
 import * as PAGES from 'constants/pages';
 import React from 'react';
 import { withRouter } from 'react-router';
+import { Link } from 'react-router-dom';
```

Comment:
There is no `Link` under `react-router`, so importing from `react-router-dom` is required here. Of course, I could import `withRouter` from there as well, but I don't know your conventions, guys, so I just made a separate line.

`<Link>` component will stand on place of each `<MenuLink>` that has prop `as={Link}` set. It allows us to utilize some declarative syntax for linking rather than pesky `onSelect...`.